.github/labeler.yml

```diff
@@ -170,3 +173,135 @@ module:sql:
   - changed-files:
       - any-glob-to-any-file:
           - solr/modules/sql/**
+
+# Any test source files
+tests:
+  - changed-files:
+      - any-glob-to-any-file:
+          - '**/src/test/**'
+          - '**/src/test-files/**'
+
+# Add category labels based on java pacakges
+# In essence, for a package org.apache.solr.foo[.xxx], assign label cat:foo
+cat:analysis:
+  - changed-files:
+      - any-glob-to-any-file:
+          - '**/org/apache/solr/analysis/**'
+
+cat:cli:
+  - changed-files:
+      - any-glob-to-any-file:
+          - '**/org/apache/solr/cli/**'
+
+cat:cloud:
+  - changed-files:
+      - any-glob-to-any-file:
+          - '**/org/apache/solr/cloud/**'
+
+cat:cluster:
+  - changed-files:
+      - any-glob-to-any-file:
+          - '**/org/apache/solr/cluster/**'
+
+cat:backup:
+  - changed-files:
+      - any-glob-to-any-file:
+          - '**/org/apache/solr/core/backup/**'
+          - '**/org/apache/solr/hdfs/backup/**'
+
+cat:core:
+  - changed-files:
+      - any-glob-to-any-file:
+          - '**/org/apache/solr/core/**'
+
+cat:filestore:
+  - changed-files:
+      - any-glob-to-any-file:
+          - '**/org/apache/solr/filestore/**'
+
+cat:handler:
+  - changed-files:
+      - any-glob-to-any-file:
+          - '**/org/apache/solr/handler/**'
+
+cat:highlight:
```

Comment:
too feature specific to warrant a category?

Reply:
Agree. I propose to fold [`highlight`, `query`, `request`, `response`, `spelling`] ==> `search`